datanode/dehistory/store/config.go

```diff
@@ -55,6 +57,17 @@ func NewDefaultConfig() Config {
 	}
 }
 
+func (c Config) GetSwarmKey(log *logging.Logger, chainID string) string {
+	swarmKey := chainID
+	if len(c.SwarmKeyOverride) > 0 {
+		swarmKey = c.SwarmKeyOverride
+		log.Infof("Using swarm key override %s as the swarm key", c.SwarmKeyOverride)
```

Comment:
structured logging preffered here as well.

Reply:
sorted - (I will try harder to remember !)